src/JcampOptions.ts

```diff
@@ -64,4 +64,12 @@ export interface JcampOptions {
    * @default ''
    */
   xyEncoding?: XYEncoding;
+  /**
+   * defines if the data type is focus on nuclear magnetic resonance
+   * @default false
+   */
+  isNMR?: boolean;
```

Comment:
Those specific options should not be here. There should be an NMRJcampOptions that extends JcampOptions